.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Should we also add `pip` here so we get updates for stuff defined in `poetry`?

Reply:
It might be worthwhile, though almost all dependencies in `pyproject.toml` are dev dependencies.

Would it be an option to merge this as-is and then add `pip` later if still desired?